WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -120,7 +122,11 @@ extension ProductFormViewModel {
     }
 
     func canViewProductInStore() -> Bool {
-        return originalProduct.product.productStatus == .publish
+        originalProduct.product.productStatus == .publish && formType == .edit
```

Comment:
Probably in the future should be better to rename `publish` to `published`. The key `publish` sounds like something that has yet to be done, and not like a status. 🌚

Reply:
yes, it is confusing to see product status is `.publish` while it means "published"! I created this issue to track this change https://github.com/woocommerce/woocommerce-ios/issues/2851 🙂